webviz_ert/data_loader/__init__.py

```diff
@@ -14,15 +14,17 @@
 def get_info(project_id: str = None) -> Mapping[str, str]:
     from ert_shared.storage.connection import get_info
 
-    return get_info(project_id)
+    info = get_info(project_id)
+    info["auth"] = info["auth"][1]
+    return info
 
 
 # these are needed to mock for testing
-def _requests_get(*args: Union[str, bytes], **kwargs: Any) -> requests.models.Response:
+def _requests_get(*args: Any, **kwargs: Any) -> requests.models.Response:
     return requests.get(*args, **kwargs)
 
 
-def _requests_post(*args: Union[str, bytes], **kwargs: Any) -> requests.models.Response:
+def _requests_post(*args: Any, **kwargs: Any) -> requests.models.Response:
```

Comment:
`Union[str, bytes,dict]`?

Reply:
fails with `error: Argument 1 to "post" has incompatible type "*Tuple[Union[str, bytes, Dict[Any, Any]], ...]"; expected "bool"  [arg-type]`

Reply:
ok, then `Any` fits the bill.